filecoin-proofs/src/api/seal.rs

```diff
@@ -1234,13 +1225,12 @@ where
     let base_tree_count = get_base_tree_count::<Tree>();
     let base_tree_leafs = leaf_count / base_tree_count;
 
-    let rows_to_discard = default_rows_to_discard(base_tree_leafs, Tree::Arity::to_usize());
     let size = get_base_tree_size::<Tree>(SectorSize(sector_size))?;
     let tree_c_config = StoreConfig {
         path: PathBuf::from(output_dir.as_ref()),
         id: CacheKey::CommCTree.to_string(),
         size: Some(size),
-        rows_to_discard,
+        rows_to_discard: 0,
```

Comment:
If TreeR has discarded rows, shouldn't also TreeC? Admittedly, I don't fully understand `rows_to_discard` and its implications

Reply:
> If TreeR has discarded rows, shouldn't also TreeC? Admittedly, I don't fully understand `rows_to_discard` and its implications

That's exactly the confusing thing, which motivated this change. The `rows_to_discard` really only ever applies to `TreeR`. Even if it's set, it would be ignored by a `TreeC`.

Reply:
To put it another way, we need to keep TreeR around for the long term for PoSt proving, where-as TreeC is removed after the initial PoRep proof usage.  The idea was that if we need to keep TreeR around -- how could we optimize (down) the amount of space that it requires on disk (i.e. compaction/level cache store).